pom.xml

```diff
@@ -50,6 +50,7 @@
                 <module>apps/structure</module>
```

Comment:
pls move it here, and add content module to all module

Reply:
Sure, will do.